CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 5.0.0-amazon.alpha.6.billing3.iap2
```

Comment:
When this PR is merged to billing-client-3-amazon, I will release it as a `5.0.0-amazon.alpha.6.billing3.iap2` version

Reply:
we probably wanna call out the new method signature here right?